CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ that users understand how the changes affect the new version.
 
 version develop
 ---------------------------
++ Make the MultiQC task suitable for use with a `final_workflow_outputs_dir`
+  so it can be used on all of Cromwell's supported backends.
++ Add BamMetrics for stats on TranscriptClean alignment.
```

Comment:
Maybe add a note on the time_minutes being added to the tasks here as well.

Reply:
I just noticed it in the recent RNA-seq PR, so I added it just as you were reviewing, but good idea indeed.